compiler/infra/BitVector.hpp

```diff
@@ -163,6 +164,24 @@ class TR_BitVector
    TR_BitVector() : _numChunks(0), _chunks(NULL), _firstChunkWithNonZero(0), _lastChunkWithNonZero(-1), _growable(growable), _region(0) { }
    TR_BitVector(TR::Region &region) : _numChunks(0), _chunks(NULL), _firstChunkWithNonZero(0), _lastChunkWithNonZero(-1), _growable(growable), _region(&region) { }
 
+   /**
+    * @brief Constructor to create a new BitVector by reading data from the serializer
+    * @param [in] serializer used for reading BitVector data
+    */
+   TR_BitVector(TR_Serializer &serializer)
+      {
+      _firstChunkWithNonZero = serializer.read<int32_t>();
+      _lastChunkWithNonZero = serializer.read<int32_t>();
+      _numChunks = serializer.read<int32_t>();
```

Comment:
perhaps you should use the multi param version of `read` to ensure that the order in which these fields are read does not change.